lib/pages/HomePages.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:flutter/material.dart';
+import 'package:hive/hive.dart';
+
+import 'CalendarTab.dart';
+import 'InsightsTab.dart';
+import 'SettingsTab.dart';
+
+class HomePages extends StatefulWidget {
+  final Box calendarBox;
+
+  const HomePages({Key? key, required this.calendarBox}) : super(key: key);
+
+  @override
+  State<HomePages> createState() => _HomePagesState();
+}
+
+class _HomePagesState extends State<HomePages> {
+  int _tabIndex = 0;
+
+  static const List<Widget> _tabContent = <Widget>[
```

Comment:
Also this is entirely just my personal preference, but since the calendar page is the primary page of interaction and is where they can log/view symptoms, I would rather put it as the center widget.

Reply:
I don't mind either way, so I'll make the change to your preference